app/views/posts/_new_comment_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_with(model:[@post, Comment.new], html: { class: 'form comment-form' }) do |form| %>
+  <fieldset>
+    <legend class="form-legend">Write your comment</legend>
+    <%= form.label :text, "Content", class: "form-label" %>
+    <%= form.text_area :text, class: "form-input form-text-area" %>
+    <%= form.submit "Save comment", class: "form-btn" %>
+  </fieldset>
+<% end %>
```

Comment:
- On your page, I can't seem to see any data added to your database showing on your app's views. if this is not the case, kindly let me contact you. 

   - The user's index page is blank, the same as other routes. Please, can you re-evaluate this? Because of this behavior, I can't further test your forms.
   
   
![image](https://github.com/Estete9/blog-app-ror/assets/59397159/a78d649a-3a66-45ac-8d4e-7c8da4de014b)

Reply:
Hello my friend, I just texted you to slack